UI/PauseScreen.cpp

```diff
@@ -385,7 +385,10 @@ void GamePauseScreen::CreateViews() {
 		I18NCategory *mm = GetI18NCategory("MainMenu");
 		rightColumnItems->Add(new Choice(mm->T("Exit")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
 	} else {
-		rightColumnItems->Add(new Choice(pa->T("Exit to menu")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
+		I18NCategory *mm = GetI18NCategory("MainMenu");
+
+		rightColumnItems->Add(new Choice(pa->T("Exit to Menu")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
+		rightColumnItems->Add(new Choice(mm->T("Exit to Desktop")))->OnClick.Handle(this, &GamePauseScreen::OnExitToDesktop);
```

Comment:
Definitely don't want this on all platforms, especially as "Desktop". Also, what's the rationale? Why are you adding this?

Also, you changed it to `OnExitToDesktop`, but it's still `OnExit` elsewhere.

-[Unknown]

Reply:
Thank you for the replies and feedback! This commit showed up on your repo by accident due to a complication with git bash. I'll be implementing the changes you suggested and proceeding with work on a separate repository that won't interfere with this one. Apologies for the inconvenience.